setup.py

```diff
@@ -76,6 +76,7 @@
 
 # Add the project-global data
 package_info['package_data'].setdefault(PACKAGENAME, [])
+package_info['package_data'][PACKAGENAME].append('*/tests/data/*')
```

Comment:
I think that the current structure, with tests in each sub package and the data for each test easily available in the tests/data subfolder of each sub package, is easier to keep track of.
I'm not against having a single data directory, but I think it would be more complicated to maintain and more easily cluttered